### PR TITLE
1.6.0

### PR DESCRIPTION
**Pull request by johnmgriffith**

### CODE REVIEW

File: icons/article-32.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M3.2 30l.002-27.798H29V30zM28 3.202H4.202L4.2 29H28zM25 6H7v1h18zm0 4H7v1h18zm0 4H7v1h18zm0 4h-8v1h8zm0 3h-8v1h8zm-3 3h-5v1h5zm-8-6v8H7v-8zm-1 1H8v6h5z"/></svg>
\ No newline at end of file
+<svg id="icons" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M25 6H7v1h18zm0 4H7v1h18zm0 4H7v1h18zm0 4h-8v1h8zm0 3h-8v1h8zm-3 3h-5v1h5zm-8-6v8H7v-8zm-1 1H8v6h5zM3 2v28h26V2zm25 27H4V3h24z"/></svg>
```
Comment (paulcpederson) [inline icons/article-32.svg]: not sure this id should be here?

**Comment by paulcpederson** [comment]
looks good, id attribute is very minor so feel free to merge

File: icons/article-32.svg (hunk 1 above)
Comment (julio8a) [reply icons/article-32.svg]: woah, good catch Paul.

**Comment by johnmgriffith** [comment]
> looks good, id attribute is very minor so feel free to merge

Regardless, we've got it fixed. Thanks for the comment.

**Pull request merged**